### PR TITLE
Ch45634/test update name

### PR DESCRIPTION
**Pull request by mitchell-gonzalez**
Added Tests for user-services - in particular testing the updateName Mutation

### Checklist

- [x] Have you fully tested your PR locally (and integration if necessary)?
~~- [ ] Have you updated any affected APIs on Insomnia?~~
~~- [ ] Have you written thorough tests for your work?~~

### CODE REVIEW

File: packages/user-service/test/providers/customer/user.provider.test.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import { stub } from 'jest-auto-stub';
```
Comment (connkat) [inline packages/user-service/test/providers/customer/user.provider.test.ts:1]: lint warnings to clean up

File: packages/user-service/test/providers/customer/user.provider.test.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+afterAll(() => {
+  jest.restoreAllMocks();
+});
+
+const user = mockUser();
+
+describe('Update Name Mutation', (): void => {
+  describe('Update First and Last Name', (): void => {
+    test('Input is undefined', async (): Promise<void> => {
```
Comment (connkat) [inline packages/user-service/test/providers/customer/user.provider.test.ts:23]: I would update the name to be more explicit. Something like `fails when input is undefined`

File: packages/user-service/test/providers/customer/user.provider.test.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      const args = {
+        userId: user.id,
+        update: { firstName: undefined, lastName: undefined },
+      };
+
+      await expect(userProvider.updateName(args)).rejects.toThrow(InvalidRequestError);
+    });
+    test('Expect Name Fields to be updated', async (): Promise<void> => {
```
Comment (connkat) [inline packages/user-service/test/providers/customer/user.provider.test.ts:32]: same here. Just something like ` passes with correct input`

**Review by connkat — CHANGES_REQUESTED** [review]
Remember to remove the `Add list of changes` from the PR template. 

You've got some lint warnings to clean up and I left some comments about naming.

**Review by connie-ho — APPROVED** [review]
Looks good :)

**Review by connkat — APPROVED** [review]
:shipit:

**Review by faddoul — APPROVED** [review]
:100:

**Review by hanjun1 — APPROVED** [review]
looks great!

**Pull request merged**